Dockerfile

```diff
@@ -1,35 +1,36 @@
 # Pull base image
-FROM python:3.6-alpine
+FROM helsinkitest/python-node:3.6-8-slim
 
 # Set environment varibles
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
```

Comment:
Not needed as the base image already defines this

Reply:
This is removed.